server/server.go

```diff
@@ -34,9 +35,11 @@ func NewServer(cfg *config.Config) *Server {
 		log.Fatal(err)
 	}
 	mw := []wish.Middleware{
-		bm.Middleware(tui.SessionHandler(ac)),
-		gm.Middleware(cfg.RepoPath, ac),
-		lm.Middleware(),
+		rm.MiddlewareWithLogger(cfg.ErrorLog,
```

Comment:
looks good but I'd put `cfg.ErrorLog` on the next line, so all the args are on their own line